app/models/note.rb

```diff
@@ -1,20 +1,57 @@
+require 'pry'
 # == Schema Information
 #
 # Table name: notes
 #
 #  id         :bigint(8)        not null, primary key
 #  title      :string           not null
 #  content    :text             not null
-#  note_type  :string           not null
+#  note_type  :integer          default("review"), not null
 #  user_id    :bigint(8)        not null
 #  created_at :datetime         not null
 #  updated_at :datetime         not null
 #
 class Note < ApplicationRecord
-  validates :title, :content, :note_type, :user, presence: true
+  validates :title, :content, :user, presence: true
+  validate :validate_content_length
 
   enum note_type: { review: 0, critique: 1 }
 
   belongs_to :user
   has_one :utility, through: :user
+
+  SHORT_THRESHOLD = { 'North Utility' => 50, 'South Utility' => 60 }.freeze
+  MEDIUM_THRESHOLD = { 'North Utility' => 100, 'South Utility' => 120 }.freeze
+
+  def validate_content_length
+    return unless content_length != 'short' && note_type == 'review'
```

Comment:
que te parece usar la notación de simbolos acá?
```suggestion
    return unless content_length != 'short' && note_type == :review
```